tests/anmykh/test_syntax_error.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+def test_syntax_error():
+    sad = {"abc": 123}
+    with pytest.raises(SyntaxError):
+        sad{"abc"}
```

Comment:
test never runs, 
you need to rewrite it

you fail on module level

Reply:
you can write test like this
it fails on import of incorrect module


```
# my_module.py

print "hello world"


# -----------------------------------
# test_errors.py
import pytest

def test_syntax_error():
    
    with pytest.raises(SyntaxError):
        import my_module
```

Reply:
fixed